tests/integration_tests/build/test_coverage.py

```diff
@@ -25,7 +25,7 @@
 if utils.is_io_uring_supported():
     COVERAGE_DICT = {"Intel": 82.92, "AMD": 82.25, "ARM": 82.42}
 else:
-    COVERAGE_DICT = {"Intel": 80.02, "AMD": 79.36, "ARM": 79.54}
+    COVERAGE_DICT = {"Intel": 79.94, "AMD": 79.36, "ARM": 79.54}
```

Comment:
I couldn't test other architectures.
Because I don't have dev environment.

If CI failed, I will fix these values.

Reply:
Just ran the CI could you please update the values respectively.

Reply:
@JonathanWoollett-Light 
updated!

But few tests that unrelated to this PR is failing.

- https://buildkite.com/firecracker/firecracker-pr/builds/181#01852ad7-e216-4653-b2b8-0f4a081b1096
- https://buildkite.com/firecracker/firecracker-pr/builds/181#01852ad7-e235-4e53-9ed4-6b10ff57f30b

Reply:
The CI failing test coverage still https://buildkite.com/firecracker/firecracker-pr/builds/199#01852ec0-e58c-4c4a-97ad-554bd4e1b740

Reply:
Reupdated!